ci-operator/step-registry/ipi/conf/vsphere/dns/ipi-conf-vsphere-dns-commands.sh

```diff
@@ -22,10 +22,17 @@ then
     export PATH="${HOME}/.local/bin:${PATH}"
     if ! command -v pip3 &> /dev/null
```

Comment:
Can you swap the logic on this `if` so that the desired path is first?
```suggestion
    if command -v pip3 &> /dev/null
```

Reply:
I see the logic swapped in the deprovision but not here.